Runtime/CoreLib/Runtime.cs

```diff
@@ -25,7 +25,8 @@ public abstract class MarshalByRefObject {
 
 	[EditorBrowsable(EditorBrowsableState.Never)]
 	[Imported]
-	[IgnoreNamespace]
+    [IgnoreNamespace]
+    [Serializable]
 	[ScriptName("Object")]
```

Comment:
With the change of KVP to a struct this is required to still allow KVP to be serializable.